packages/tracing/package.json

```diff
@@ -30,29 +30,8 @@
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
   "devDependencies": {
-    "@aws-cdk/aws-dynamodb": "^1.145.0",
-    "@aws-cdk/aws-lambda-nodejs": "^1.145.0",
-    "@aws-cdk/core": "^1.145.0",
     "@aws-sdk/client-dynamodb": "^3.52.0",
-    "@commitlint/cli": "^16.2.1",
-    "@middy/core": "^2.5.6",
-    "@types/aws-lambda": "^8.10.72",
-    "@types/jest": "^27.4.0",
-    "@types/node": "^17.0.8",
-    "@typescript-eslint/eslint-plugin": "^5.12.1",
-    "@typescript-eslint/parser": "^5.12.1",
-    "aws-cdk": "^1.145.0",
-    "aws-sdk": "^2.1078.0",
-    "esbuild": "^0.14.23",
-    "eslint": "^8.3.0",
-    "eslint-import-resolver-node": "^0.3.6",
-    "eslint-import-resolver-typescript": "^2.5.0",
-    "eslint-plugin-import": "^2.25.3",
-    "jest": "^27.5.1",
-    "jest-runner-groups": "^2.1.0",
-    "ts-jest": "^27.1.3",
-    "ts-node": "^10.0.0",
-    "typescript": "^4.1.3"
+    "@middy/core": "^2.5.6"
```

Comment:
Question: can this be moved in the root package.json? same as for metrics and logger

Reply:
Yes it can, in this case I deliberately decided to leave it because it's the only one that historically was a runtime dependency, but on second thought it might belong with the others as I find it unlikely that customers who install the package will dig into their `node_modules` to check which version to install.
I'll move it, in the future we should consider making explicit which version we support in the docs maybe, what do you think?

Reply:
I agree